crates/l2/contracts/src/l2/CommonBridgeL2.sol

```diff
@@ -19,8 +21,31 @@ contract CommonBridgeL2 is ICommonBridgeL2 {
         require(success, "Failed to burn Ether");
 
         IL1Messenger(L1_MESSENGER).sendMessageToL1(keccak256(abi.encodePacked(
+            AssetType.ETH,
```

Comment:
Let's use the same format for both ERC20 and ETH transfers, but using a random address (like `0xeeeeeeee...eeeeeeeee`) for the L1 and L2 tokens in case it's ETH. This makes both cases almost the same, apart from some `if`s.